ci/do_ci.sh

```diff
@@ -46,6 +76,15 @@ case $build_target in
   "test")
     test_salvo
     ;;
+  "check_format")
+    check_format
+    ;;
+  "fix_format")
+    fix_format
+    ;;
+  "coverage")
+    coverage
+    ;;
   *)
     ;;
```

Comment:
Should we add a helpful error message here that will be printed out when an invalid argument will be provided?

Example:

https://github.com/envoyproxy/nighthawk/blob/2256da19d138866ca82adff2de7c5a5071cb430e/ci/do_ci.sh#L322-L325

Reply:
It's a good point, added it.